### PR TITLE
89 player name

### LINKED ISSUE #89: Player name
A Player should have the ability to name his/her character

### PR DESCRIPTION
**Pull request by MKerkhof**
Will close #89

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/group9/gui/LoadGameScene.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    public static ListView<String> getList() {
+        return list;
+    }
+
+
+    /** Creating the list. */
+        static private ObservableList<String> players =
+                FXCollections.observableArrayList();
+    /** Creating the listview used to display the list. */
+        static private ListView<String> list = createList(2, 16);
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/gui/LoadGameScene.java]: Package visibility is not allowed in Checkstyle.

File: src/main/java/nl/tudelft/ti2206/group9/gui/MainMenuScene.java [hunk 2]
```diff
@@ -26,8 +30,12 @@
 		/** Start button. */
 		START,
 		/** Settings button. */
-		SETTINGS
+		SETTINGS,
+		/** Load Game button. */
+		LOAD
 	}
+	/** The input field for the name of the player. */
+    final static TextField input = createTextField("PLAYER NAME", 2, 22);
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/gui/MainMenuScene.java]: This line is breaking about three Checkstyle rules at once.

File: src/main/java/nl/tudelft/ti2206/group9/gui/MenuScene.java [hunk 3]
```diff
@@ -1,15 +1,15 @@
 package nl.tudelft.ti2206.group9.gui;
 
+import javafx.scene.control.*;
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/gui/MenuScene.java]: Star imports not allowed.

**Comment by mpsijm** [comment]
Please pull the master in your branch and resolve all SAT warnings before merging this branch. The functionality looks good though :)

<review,layout

**Comment by MathiasMeuleman** [comment]
As @mpsijm said, merging master is imortant! Also I found a number of checkstyle warnings. Furthermore I think he covered about all that can be said.

<review

**Comment by mpsijm** [comment]
Okay, I fixed some more stuff for you. Once @mdingjan agrees, he can merge!

<review,layout,documentation

**Comment by mdingjan** [comment]
I peer reviewed this branch with Maarten. There are no checkstyle warnings left.

Will merge!

<review

**Pull request merged**